keyboards/helix/pico/keymaps/default/readme_jp.md

```diff
@@ -131,6 +105,48 @@ RBG Underglow や RGBバックライトの輝度を抑えて、iPad, iPhone に
 IOS_DEVICE_ENABLE = no      # connect to IOS device (iPad,iPhone)
 ```
 
+## コンパイルの仕方
+
+コンパイルは、qmk_firmware のトップディレクトリで行います。
+
+```
+$ cd qmk_firmware
+```
+qmk_firmwareでは各キーボードのコンパイルは、`<キーボード名>:<キーマップ名>`という指定で行います。
+
+```
+$ make helix/pico:default
+```
+
+キーボードへの書き込みまで同時に行うには下記のように`:avrdude`を付けます。
+
+```
+$ make helix/pico:default:avrdude
```

Comment:
All the `:avrdude` can be replaced with `:flash`.  It should autodetect if the bootloader is set correctly.

Reply:
Good, 'flash' is a meaningful target name.